packages/task/src/browser/task-service.ts

```diff
@@ -32,6 +32,8 @@ import { ProvidedTaskConfigurations } from './provided-task-configurations';
 import { Range } from 'vscode-languageserver-types';
 import URI from '@theia/core/lib/common/uri';
 
+const deepmerge: (args: object[]) => object = require('deepmerge').default.all;
```

Comment:
my curiosity: would 
``` ts
import { default.all as deepmerge } from 'deepmerge';
```
work?

Reply:
Unfortunately I failed to get it work via import.
Your proposal 
```
import { default.all as deepmerge } from 'deepmerge';
```
also failed with the following error: 
```
lerna ERR! [compile] src/browser/task-service.ts(36,10): error TS1003: Identifier expected.
```